libcxx/test/tools/clang_tidy_checks/CMakeLists.txt

```diff
@@ -64,6 +64,26 @@ if(NOT HAS_CLANG_TIDY_HEADERS)
                  "clang-tidy headers are not present.")
   return()
 endif()
+
+# The clangTidy plugin uses C++20, when using stdlibc++ verify the version is correct.
```

Comment:
```suggestion
# The clangTidy plugin uses C++20, so ensure that we support C++20 when using libstdc++.
# This is required because some versions of libstdc++ used as a system library on build platforms
# we support do not support C++20 yet.
```